com.liferay.blade.eclipse.provider/src/com/liferay/blade/eclipse/provider/JavaFileJDT.java

```diff
@@ -168,6 +168,8 @@ public boolean visit(TypeDeclaration node) {
 	public SearchResult findImport(final String importName) {
 		final List<SearchResult> searchResults = new ArrayList<>();
 
+		setFile(_file);
```

Comment:
Set file should not be just called for "findImport" what about other findXXX commands, if the file is not set, it should always be set priort to use.  I don't think we need "setFile" called at all.